python/common_modules_cff.py

```diff
@@ -117,6 +117,38 @@
     ) * probeMuonsIsoValueMaps
 )
 
+muonMiniIsoCharged = cms.EDProducer("MuonMiniIso",
+    probes = cms.InputTag("probeMuons"),
+    pfCandidates = cms.InputTag("pfAllChargedHadronsPFBRECO"),
+    dRCandProbeVeto = cms.double(0.0001),
+    CandPtThreshold = cms.double(0.0),
+    CandPdgId = cms.string('h'), # 'h','gamma','h0' or leave empty for all
+)
+
+muonMiniIsoPUCharged = cms.EDProducer("MuonMiniIso",
+    probes = cms.InputTag("probeMuons"),
+    pfCandidates = cms.InputTag("pfPileUpAllChargedParticlesPFBRECO"),
+    dRCandProbeVeto = cms.double(0.0001),
+    CandPtThreshold = cms.double(0.0),
+    CandPdgId = cms.string('h'), # 'h','gamma','h0' or leave empty for all
```

Comment:
I'd rather you leave the string empty, i.e. use whatever `pfPileUpAllChargedParticlesPFBRECO ` provides.
In fact, for none of the 4 modules need to apply any selection on the pdgId, as it's always alread done  upstream